tests/clients/logger/test_logger.py

```diff
@@ -0,0 +1,44 @@
+from typing import Generator
+from io import StringIO
+
+import pytest
+
+from mlrun.clients.logger import create_logger, LoggerFormatterEnum, Logger
+
+
+@pytest.fixture(scope="session")
+def stream() -> Generator:
+    io_stream = StringIO()
+    try:
+        yield io_stream
+    finally:
+        io_stream.close()
+
+
+@pytest.fixture(params=list(LoggerFormatterEnum.__members__))
```

Comment:
I don't think the `.__members__` is needed

Reply:
it...is